proto/replicationdata.proto

```diff
@@ -25,9 +25,8 @@ package replicationdata;
 // flavor-specific command and parsed into a Position and fields.
 message Status {
   string position = 1;
-  // 2 used to be io_thread_running. Instead io_state is used.
-  // 3 used to be sql_thread_running. Instead sql_state is used.
-  reserved 2, 3;
+  bool io_thread_running = 2;
```

Comment:
This change was done just to stop `protojson` returning unmarshalling errors. Nothing actually uses this value